docs/en/sql-reference/statements/update.md

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Also need to update docs in https://github.com/ClickHouse/clickhouse-docs/blob/main/docs/managing-data/updating-data/index.md and https://github.com/ClickHouse/clickhouse-docs/blob/main/docs/guides/developer/lightweight-update.md